### PR TITLE
add 2 references

### PR DESCRIPTION
**Pull request by jiemdev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -832,7 +832,7 @@ Europen Conf. Computer Vision (ECCV), 2022. [YouTube](https://youtu.be/nUb2ZRPdb
 - <a name="Shiba22spl"></a>Shiba, S., Aoki, Y., Gallego, G.,  
 *[Fast Event-Based Optical Flow Estimation by Triplet Matching](https://doi.org/10.1109/LSP.2023.3234800)*,  
 IEEE Signal Process. Lett. (SPL), 29:2712-2716, 2022. [PDF](https://arxiv.org/pdf/2212.12218).
-- <a name="Brebion21tits"></a>Brebion, V., Moreau, J., Davoine, F.,  
+- <a name="Brebion21tits"></a>Brebion, V., , J., Davoine, F.,
```
Comment (guillermogb) [inline README.md:835]: why is this last name deleted?

File: README.md [hunk 2]
```diff
@@ -1180,7 +1180,7 @@ IEEE Robotics and Automation Letters (RA-L), 2021. [PDF](http://rpg.ifi.uzh.ch/d
 - <a name="Muglikar23cvpr"></a>Muglikar, M., Bauersfeld, L., Moeys, D., Scaramuzza, D.,  
 *[Event-based Shape from Polarization](https://arxiv.org/abs/2301.06855)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2023. [PDF](https://rpg.ifi.uzh.ch/docs/CVPR23_Muglikar.pdf), [YouTube](https://youtu.be/sF3Ue2Zkpec), [code](https://github.com/uzh-rpg/ESfP).  
-- <a name="Brebion23scia"></a>Brebion, V., Moreau, J., Davoine, F.,  
+- <a name="Brebion23scia"></a>Brebion, V., , J., Davoine, F.,
```
Comment (guillermogb) [inline README.md:1183]: why is this last name deleted?

File: README.md [hunk 3]
```diff
@@ -1790,6 +1790,8 @@ IEEE Trans. Neural Netw. Learn. Syst. (TNNLS), 2021.  [Project page](https://sit
 - <a name="Wang23arxiva"></a>Wang, Z., Guo, J., Daniilidis, K.,  
 *[Un-EvMoSeg: Unsupervised Event-based Independent Motion Segmentation](https://arxiv.org/pdf/2312.00114.pdf)*,  
 arxiv, 2023.  [Project page](https://www.cis.upenn.edu/~ziyunw/un_evmoseg/).
+- <a name="Jiang24visapp"></a>Jiang, C., , J., Davoine, F. *[Event-based Semantic-aided Motion Segmentation](https://doi.org/10.5220/0012308100003660)*,
```
Comment (guillermogb) [inline README.md:1793]: missing last name of 2nd author

File: README.md [hunk 4]
```diff
@@ -1790,6 +1790,8 @@ IEEE Trans. Neural Netw. Learn. Syst. (TNNLS), 2021.  [Project page](https://sit
 - <a name="Wang23arxiva"></a>Wang, Z., Guo, J., Daniilidis, K.,  
 *[Un-EvMoSeg: Unsupervised Event-based Independent Motion Segmentation](https://arxiv.org/pdf/2312.00114.pdf)*,  
 arxiv, 2023.  [Project page](https://www.cis.upenn.edu/~ziyunw/un_evmoseg/).
+- <a name="Jiang24visapp"></a>Jiang, C., , J., Davoine, F. *[Event-based Semantic-aided Motion Segmentation](https://doi.org/10.5220/0012308100003660)*,  
+Proceedings of the 19th International Joint Conference on Computer Vision, Imaging and Computer Graphics Theory and Applications (VISAPP), 2024.
```
Comment (guillermogb) [inline README.md:1794]: Int. Joint Conf. on Computer Vision, Imaging and Computer Graphics Theory and Applications (VISAPP) 

(see other VISAPP 2020 entry)

File: README.md [hunk 5]
```diff
@@ -3022,6 +3024,8 @@ IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2021. [Video](https:/
 - <a name="Bao23arxiv"></a>Bao, Y., Sun, L., Ma, Y., Gu, D., Wang, K.,  
 *[Improving Fast Auto-Focus with Event Polarity](https://arxiv.org/abs/2303.08611)*,  
 Arxiv, 2023.
+- <a name="Cocheteux24cvprw"></a>Cocheteux, M., Moreau, J., Davoine, F., *[MULi-Ev: Maintaining Unperturbed LiDAR-Event Calibration](https://openaccess.thecvf.com/content/CVPR2024W/WAD/html/Cocheteux_MULi-Ev_Maintaining_Unperturbed_LiDAR-Event_Calibration_CVPRW_2024_paper.html)*,
```
Comment (guillermogb) [inline README.md:3027]: In 3 lines please; the preview shows this entry in 2-line format only.

**Review by guillermogb — CHANGES_REQUESTED** [review]
thanks

File: README.md (hunk 1 above)
Comment (jiemdev) [reply README.md:835]: I did an error, sorry

File: README.md [hunk 6]
```diff
@@ -3022,6 +3025,9 @@ IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2021. [Video](https:/
 - <a name="Bao23arxiv"></a>Bao, Y., Sun, L., Ma, Y., Gu, D., Wang, K.,  
 *[Improving Fast Auto-Focus with Event Polarity](https://arxiv.org/abs/2303.08611)*,  
 Arxiv, 2023.
+- <a name="Cocheteux24cvprw"></a>Cocheteux, M., Moreau, J., Davoine, F.,  
+*[MULi-Ev: Maintaining Unperturbed LiDAR-Event Calibration](https://openaccess.thecvf.com/content/CVPR2024W/WAD/html/Cocheteux_MULi-Ev_Maintaining_Unperturbed_LiDAR-Event_Calibration_CVPRW_2024_paper.html)*,  
+CVPR Workshop on Autonomous Driving, 2024.
```
Comment (guillermogb) [inline README.md:3030]: IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2024
See 3 entries above, please.

Comment (guillermogb) [reply README.md:3030]: we're almost there

**Review by guillermogb — CHANGES_REQUESTED** [review]

**Review by guillermogb — APPROVED** [review]

**Pull request merged**